curriculum/challenges/english/25-front-end-development/lecture-working-with-relative-and-absolute-units/672bb83c3a9906945536cff2.md

```diff
@@ -10,6 +10,53 @@ dashedName: what-are-vh-and-vw-units
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+What are vh and vw units, and when should you use them?
+
+In CSS, vh and vw are viewport-relative units that allow you to size elements based on the dimensions of the browser window. These units are particularly useful for creating responsive designs that adapt to different screen sizes.
+
+vh stands for "viewport height," and 1vh is equal to 1% of the viewport's height.
+
+Similarly, vw stands for "viewport width," and 1vw is equal to 1% of the viewport's width. 
+
+This means that if you set an element's height to 100vh, it will occupy the full height of the viewport, regardless of the actual pixel dimensions of the device.
+
+These units are especially handy when you want to create full-screen layouts or elements that maintain a specific proportion of the screen. 
+
+For example, you might want to use them to create a hero section that always fills the entire screen:
+
+```css
+.hero {
+  height: 100vh;
+  width: 100vw;
+}
+```
+
+This CSS ensures that the hero section will always be exactly the size of the viewport, regardless of the device's screen size.
+
+vh and vw units can also be used for typography to create responsive text sizes. For instance:
+
+```css
+h1 {
+  font-size: 5vw;
+}
+```
+
+This will set the font size of h1 elements to 5% of the viewport width, allowing the text to scale smoothly with the browser window size.
```

Comment:
```suggestion
This will set the font size of `h1` elements to 5% of the viewport width, allowing the text to scale smoothly with the browser window size.
```